File: AgendaContatos/app/src/main/java/com/example/agendacontatos/MainActivity.java
package com.example.agendacontatos;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        EditText txtNome = findViewById(R.id.txtNome);
        EditText txtEmail = findViewById(R.id.txtEmail);
        EditText txtTel = findViewById(R.id.txtTel);
        Button bt_gravar = findViewById(R.id.button);


        bt_gravar.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {

                String nome = txtNome.getText().toString();
                String email = txtEmail.getText().toString();
                String telefone = txtTel.getText().toString();

                Log.d("Contato", "Nome: " + nome + ", Email: " + email + ", Telefone: " + telefone);
            }
        });
    }
}

File: aula-27-03/app/src/main/java/com/example/aula2703/MainActivity.kt
package com.example.aula2703

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

public class MainActivity : AppCompatActivity() {
    override fun onCreate(bundle : Bundle?) {
        super.onCreate(bundle)
        setContentView(R.layout.activity_main)

        val btnConverter = findViewById<Button>(R.id.btnConverter)
        val editMoeda = findViewById<EditText>(R.id.editMoeda)
        val editCotacao = findViewById<EditText>(R.id.editCotacao)
        val txtResultado = findViewById<TextView>(R.id.txtResultado)

        btnConverter.setOnClickListener {
            val moeda = editMoeda.text.toString().toFloat();
            val cocacao = editCotacao.text.toString().toFloat();

            val resultado = moeda * cocacao

            txtResultado.text = "Resultado: ${resultado}"
        }
    }
}